### PR TITLE
docs: update triage manager tasks in contributing.md

### PR DESCRIPTION
**Pull request by cdrage**
docs: update triage manager tasks in contributing.md

### What does this PR do?

Adds more responsibilities / tasks for the triage manager

### Screenshot/screencast of this PR

N/A

### What issues does this PR fix or reference?

Discussed in sprint planning call.

### How to test this PR?

N/A

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -362,12 +362,11 @@ See our [EXTENSIONS.md](/EXTENSIONS.md) document for more details.
...
+Each sprint a new "Triage manager" will be assigned.
 
-Ideally complete the task every morning:
-
-1. [View all the current triage labeled issues.](https://github.com/containers/podman-desktop/issues?q=is%3Aopen+is%3Aissue+label%3Astatus%2Fneed-triage)
-2. Add the appropriate [area label](https://github.com/containers/podman-desktop/labels?q=area%2F) that matches the issue. You can use "bulk edit" by clicking the checkmark of the issues and using the label dropdown selection.
-3. Remove the `status/need-triage` label.
\ No newline at end of file
+Your responsibilities include:
+* Assigning the [status/need-triage](https://github.com/containers/podman-desktop/issues?q=is%3Aopen+is%3Aissue+label%3Astatus%2Fneed-triage) label to new issues. As a maintainer, you will need to categorize these issues under the correct [area labels](https://github.com/containers/podman-desktop/labels?q=area%2F). Once categorized, remove the `status/need-triage` label and apply the appropriate area label.
```
Comment (benoitf) [inline CONTRIBUTING.md]: AFAIK it's not the responsibility of the triager to assign `need-triage` but to review those with this label

Comment (cdrage) [reply CONTRIBUTING.md]: Woops, It should of been "Reviewing" not assigning.

**Review by benoitf — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -362,12 +362,11 @@ See our [EXTENSIONS.md](/EXTENSIONS.md) document for more details.
...
-3. Remove the `status/need-triage` label.
\ No newline at end of file
+Your responsibilities include:
+* Reviewing the [status/need-triage](https://github.com/containers/podman-desktop/issues?q=is%3Aopen+is%3Aissue+label%3Astatus%2Fneed-triage) label on new issues. As a maintainer, you will need to categorize these issues under the correct [area labels](https://github.com/containers/podman-desktop/labels?q=area%2F). Once categorized, remove the `status/need-triage` label and apply the appropriate area label.
+* Evaluating the severity of new issues. If an issue is classified as "critical" or "high priority" and requires immediate attention, tag a maintainer in the issue and notify them via the public community channel.
+* Identifying issues that are simple to resolve and marking them as "good first issue," thereby encouraging newcomers to contribute to the project. If the issue is not resolved within a maximum of three weeks, remove the label and assign it to a maintainer.
```
Comment (deboer-tim) [inline CONTRIBUTING.md]: Do we always want to assign after 3 weeks? Some are good to just knock out and close, but it seems like others might be good first issues but not priority.

**Review by deboer-tim — COMMENTED** [review]
I thought on the call we talked about the triage manager also taking a pass at stale PR cleanup (at least once per sprint)? e.g. making sure community PRs don't get stuck or asking stale PR owners if they need help

File: CONTRIBUTING.md (hunk 2 above)
Comment (benoitf) [reply CONTRIBUTING.md]: we can keep them unassigned

Comment (benoitf) [inline CONTRIBUTING.md]:
```suggestion
* Identifying issues that are simple to resolve and marking them as "good first issue," thereby encouraging newcomers to contribute to the project.
```

**Comment by cdrage** [comment]
> I thought on the call we talked about the triage manager also taking a pass at stale PR cleanup (at least once per sprint)? e.g. making sure community PRs don't get stuck or asking stale PR owners if they need help

Looks like I didn't push to this branch correctly, I've updated the PR to include the other responsibility.

**Review by benoitf — APPROVED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -362,12 +362,12 @@ See our [EXTENSIONS.md](/EXTENSIONS.md) document for more details.
...
-3. Remove the `status/need-triage` label.
\ No newline at end of file
+Your responsibilities include:
+* Reviewing the [status/need-triage](https://github.com/containers/podman-desktop/issues?q=is%3Aopen+is%3Aissue+label%3Astatus%2Fneed-triage) label on new issues. As a maintainer, you will need to categorize these issues under the correct [area labels](https://github.com/containers/podman-desktop/labels?q=area%2F). Once categorized, remove the `status/need-triage` label and apply the appropriate area label.
+* Evaluating the severity of new issues. If an issue is classified as "critical" or "high priority" and requires immediate attention, tag a maintainer in the issue and notify them via the public community channel.
+* Identifying issues that are simple to resolve and marking them as "good first issue," thereby encouraging newcomers to contribute to the project.
+* Evaluating any stale / lingering pull requests and pinging the respective contributors. If the pull request has been opened for an extensive amount of time, collaborate with the contributor / push any changes required to get it merged in. If there is no communication / the pull request is stale, close them.
```
Comment (deboer-tim) [inline CONTRIBUTING.md]: > collaborate with the contributor / push any changes

I wouldn't even say this is necessarily the triager's responsibility as much as just making sure _someone_ does, but I'm fine as-is.

LGTM

**Review by deboer-tim — APPROVED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (cdrage) [reply CONTRIBUTING.md]: Changed the wording so it says "someone" rather than putting it on the triager's plate.

**Pull request merged**